### PR TITLE
Redux for experiments table drag and drop

### PR DESCRIPTION
**Pull request by sroy3**
As this reducer already existed, it was easy to remove the `DragDropContext` and just use the reducer. Things seem to work just as they did for drag and drop. @wolmir can you test and tell me if I missed anything. All tests were passing, so it should be fine, but we never know...

Starting work for #2103

### CODE REVIEW

**Review by julieg18 — APPROVED** [review]
Looks good to me!

**Comment by wolmir** [comment]
Sorry @sroy3 I'll test it now

**Review by wolmir — APPROVED** [review]
Works great, thanks!

File: webview/src/stories/Table.stories.tsx [hunk 1]
```diff
@@ -196,9 +203,11 @@ WithNoSortsOrFilters.args = {
...
 export const Scrolled: ComponentStory<typeof Experiments> = ({ tableData }) => {
   return (
-    <div style={{ height: 400, width: 600 }}>
-      <Experiments tableData={tableData} />
-    </div>
+    <Provider store={configureStore({ reducer: experimentsReducers })}>
+      <div style={{ height: 400, width: 600 }}>
+        <Experiments tableData={tableData} />
+      </div>
+    </Provider>
```
Comment (rogermparent) [inline webview/src/stories/Table.stories.tsx:210]: Maybe `Provider` could be added with a [decorator](https://storybook.js.org/docs/react/writing-stories/decorators)?

Comment (sroy3) [reply webview/src/stories/Table.stories.tsx:210]: That is a good idea, I'll add this in my follow up PR

**Pull request merged**